docs/network-policy-recommendation.md

```diff
@@ -0,0 +1,148 @@
+# Network Policy Recommendation
+
+## Table of Contents
+
+<!-- toc -->
+- [Introduction](#introduction)
+- [Deployment](#deployment)
+- [Perform NetworkPolicy Recommendation](#perform-networkpolicy-recommendation)
+  - [Run a policy recommendation job](#run-a-policy-recommendation-job)
+  - [Check the status of a policy recommendation job](#check-the-status-of-a-policy-recommendation-job)
+  - [Retrieve the result of a policy recommendation job](#retrieve-the-result-of-a-policy-recommendation-job)
+<!-- /toc -->
+
+## Introduction
+
+Theia NetworkPolicy Recommendation recommends the NetworkPolicy configuration to secure Kubernetes
+network and applications. It analyzes the network flows collected by
+[Grafana Flow Collector](network-flow-visibility.md#grafana-flow-collector) to generate
+[Kubernetes NetworkPolicies](https://kubernetes.io/docs/concepts/services-networking/network-policies/)
+or [Antrea NetworkPolicies](https://github.com/antrea-io/antrea/blob/main/docs/antrea-network-policy.md).
+The goal of this feature is to aid the cluster admin or developers to secure
```

Comment:
Consider rephrase:

This feature assists cluster administrators and app developers in securing their applications according to Zero Trust principles

Reply:
Thanks, addressed.